app/controllers/v0/forms_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module V0
+  class FormsController < ApplicationController
+    include ActionView::Helpers::SanitizeHelper
+
+    skip_before_action :authenticate
+
+    def index
+      response = Forms::Client.new(query).get_all
+      render json: response.body,
+             status: response.status
+    end
+
+    def query_params
+      params.permit(:query)
+    end
+
+    def query
+      sanitize query_params['query']
```

Comment:
I *think* you might just want this to be a one-liner if you can -- i think the multiple methods with the naming here are more confusing rather than lesss

Reply:
```
def search_params
	query = params.permit(:query)
	sanitize query
end
```
or a one-liner.